### PR TITLE
Reset database

### PR DESCRIPTION
**Pull request by bourque**
This PR adds a `reset_database.py` module, which will reset the database given in the `connection_string` key in the `config.json` config file.  This can be used to easily reset the development database.

### CODE REVIEW

**Comment by bourque** [comment]
@hover2pi You've once thought about a database before, mind reviewing this one?

File: jwql/database/database_interface.py [hunk 1]
```diff
@@ -208,7 +208,7 @@ class Monitor(base):
     monitor_name = Column(String(), nullable=False)
     start_time = Column(DateTime, nullable=False)
     end_time = Column(DateTime, nullable=True)
-    status = Column(Enum('SUCESS', 'FAILURE'), nullable=True)
+    status = Column(Enum('SUCESS', 'FAILURE', name='monitor_status'), nullable=True)
```
Comment (bourque) [inline jwql/database/database_interface.py]: Note that this is a small bug fix that doesn't have anything to do with `reset_database.py`

**Comment by laurenmarietta** [comment]
(Closes #318.)

**Comment by bourque** [comment]
@laurenmarietta I envision that we could use this to completely wipe out and start from scratch with our development database, particularly in the case where we need to create new tables, or restructure existing tables, or have made mistakes in the kind of data we are storing.

This made me realize though that someone could use this to reset the production database, which would not be good.  I should add something in the code here that doesn't allow one to do that.

**Comment by laurenmarietta** [comment]
Yes, I think that is a great idea! I was afraid to test it myself mainly because I was worried I would accidentally delete something I shouldn't... Including some checks would be helpful.

**Pull request merged**